batchupload/uploader.py

```diff
@@ -19,7 +20,7 @@ def upload_single_file(file_name, media_file, text, target_site,
     Upload a single file in chunks.
 
     @param file_name: sanitized filename to use for upload
-    @param media_file: path to media file to upload
+    @param media_file: path or URL to media file to upload
     @param text: file description page
     @param target_site: pywikibot.Site object to which file should be uploaded
     @param chunk_size: Size of chunks (in MB) in which to upload file
```

Comment:
I'm kinda thinking aloud here, and I don't really know the ins and outs of chunked uploads, so this is not super important... Right now, the two available options for chunk size are the hard-coded 5 MB and whatever the user inputs. I'm thinking there should be a possibility for the script to automatically adjust the chunk size based on the file size.